src/wwwroot/partials/settings/billing.html

```diff
@@ -103,54 +95,44 @@ <h3 class="special">{{ 'payment_method_title' | translate }}</h3>
               <label class="special uppercase" for="cardHolder">{{'card_holder_label' | translate}}</label>
             </div>
             <div class="flex three flex--wrap">
-              <input id="cardHolder" name="cardHolder" ng-model="vm.cardHolder" ng-class="(vm.submitted || form.cardHolder.$dirty) && form.cardHolder.$invalid ? 'error' : ''" class="input input--large" required />
-              <div class="width--full animate-error-container" ng-class="(vm.submitted || form.cardHolder.$dirty) && form.cardHolder.$invalid ? 'show' : 'hide'">
-                <span class="input--error" ng-show="(vm.submitted || form.cardHolder.$dirty) && form.cardHolder.$error.required"><i class="arrow--up"></i>{{'validation_error_required' | translate}}</span>
-              </div>
+              <input id="cardHolder" name="cardHolder" ng-model="vm.cardHolder" class="input input--large" validation-errors-fluid  required />              
             </div>
           </div>
           <div class="item flex">
             <div class="flex one flex--vertically-start">
               <label class="special uppercase" for="cardNumber">{{'card_number_label' | translate}}</label>
             </div>
-          <div class="flex three credit-card flex--wrap" ng-class="vm.cc.brand">
-              <input id="cc.number" ng-model="vm.cc.number" name="cardNumber" autocomplete="off" ui-mask="{{vm.cc.mask}}" ui-options="vm.maskOptions" ng-class="(vm.submitted || form.cardNumber.$dirty) && form.cardNumber.$invalid ? 'error' : ''" class="input input--large" required/>
-            <svg class="icon icon-visa">
-              <use xlink:href="/images/sprite.svg#doppler-icon-visa-card"></use>
-            </svg>
-            <svg class="icon icon-master">
-              <use xlink:href="/images/sprite.svg#doppler-icon-master-card"></use>
-            </svg>
-            <svg class="icon icon-amex">
-              <use xlink:href="/images/sprite.svg#doppler-icon-american-card"></use>
-            </svg>
-              <div class="width--full animate-error-container" ng-class="(vm.submitted || form.cardNumber.$dirty) && form.cardNumber.$invalid ? 'show' : 'hide'">
-                <span class="input--error" ng-show="(vm.submitted || form.cardNumber.$dirty) && form.cardNumber.$error.required || form.cardNumber.$error.mask"><i class="arrow--up"></i>{{'validation_error_required' | translate}}</span>
-                <span class="input--error" ng-show="(vm.submitted || form.cardNumber.$dirty) && form.cardNumber.$error.ilegal_number && !form.cardNumber.$error.required && ! form.cardNumber.$error.mask"><i class="arrow--up"></i>{{'validation_error_invalid_card_number' | translate}}</span>
```

Comment:
😍 😍 😍 😍  This is the best one! you have removed two ugly elements, not just one!